db/migrate/20220615150831_add_column_to_greetings.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToGreetings < ActiveRecord::Migration[7.0]
+  def change
+    add_column :greetings, :greeting, :string
+  end
+end
```

Comment:
- [ ] Please make sure your migration is in the right way. First, you need to create a table `greetings` to add any column or data. Please fix that otherwise migration and seeding are giving errors.

![Screenshot from 2022-06-15 02-57-24](https://user-images.githubusercontent.com/26767383/173693151-e802f0e1-388c-4aaa-930f-53b560cc9f45.png)

Reply:
resolved